src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -448,6 +446,48 @@ def lease(self, _event=None):  # pylint: disable=arguments-differ
     track_task_end()
 
 
+class PostprocessPubSubTask(PubSubTask):
+  """A postprocess task received over pub/sub."""
+
+  def __init__(self,
+               output_url_argument,
+               pubsub_message,
+               is_command_override=False):
+    command = 'postprocess'
+    job_type = 'none'
+    eta = None
+    high_end = False
+    grandparent_class = super(PubSubTask, self)
```

Comment:
Why grandparent_class? Can we restructure the classes a bit instead a bit better to avoid this (or just rely on super() directly? )